pkg/kubelet/cm/qos_container_manager_linux_test.go

```diff
@@ -310,7 +310,6 @@ func TestQosContainerManagerImpl_Start(t *testing.T) {
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
 			mockCtrl := gomock.NewController(t)
-			defer mockCtrl.Finish()
```

Comment:
Delete this line according to the information provided by @endocrimes at https://github.com/kubernetes/kubernetes/pull/109746#discussion_r864559039.
Thank you!